script.js

```diff
@@ -0,0 +1,259 @@
+];
+
+const wrapper = document.createElement('div');
+wrapper.id = 'wrapper';
+document.body.append(wrapper);
+const textarea = document.createElement('textarea');
+textarea.readOnly = 'readonly';
+textarea.maxlength = '500';
+textarea.id = 'display';
+wrapper.append(textarea);
+const display = document.querySelector('#display');
+const keyboards = document.createElement('div');
+keyboards.className = 'keyboard';
+keyboards.id = 'keyboard';
+wrapper.append(keyboards);
+const text = document.createElement('p');
+text.innerText = 'Ctrl + Alt  смена языка   Del - очистить поле';
+wrapper.append(text);
+
+
+for (let i = 0, j = 0, n = 0, l = 0;
+  i < 63, j < CODE.length, n < RUS.length, l < RUS_CAPS.length;
+  i += 1, j += 1, n += 1, l += 1) {
+  const keys = document.createElement('div');
+  keys.classList.add('key');
+  keys.id = 'key';
+  keys.setAttribute('data-code', CODE[j]);
+  keys.setAttribute('data-ru', RUS[n]);
+  keys.setAttribute('data-ru-caps', RUS_CAPS[l]);
+  keys.innerHTML = `<span class="print en" > ${RUS[n]} </span><span class="enCap hidden print">${RUS_CAPS[l]}</span>`;
+  keyboards.append(keys);
+}
+keyboards.children[13].classList.add('backspace');
+keyboards.children[14].classList.add('tab');
+keyboards.children[28].classList.add('del');
+keyboards.children[29].classList.add('CapsLock');
+keyboards.children[41].classList.add('Enter');
+keyboards.children[42].classList.add('ShiftLeft');
+keyboards.children[53].classList.add('ArrowUp');
+keyboards.children[54].classList.add('ShiftRight');
+keyboards.children[55].classList.add('ControlLeft');
+keyboards.children[56].classList.add('Meta');
+keyboards.children[57].classList.add('AltLeft');
+keyboards.children[58].classList.add('Space');
+keyboards.children[59].classList.add('AltRight');
+keyboards.children[63].classList.add('ControlRight');
+keyboards.children[60].classList.add('ArrowLeft');
+keyboards.children[61].classList.add('ArrowDown');
+keyboards.children[62].classList.add('ArrowRight');
+
+window.onload = function () {
+  const s = localStorage.getItem('isEng');
+  console.log(s);
+  if (localStorage.length === 0) {
+    return eng();
+  } if (s === 1) {
+    return rus();
+  }
+  return rus();
+};
+function eng() {
+  const span = document.querySelectorAll('.print');
+  span.innerText = '';
+  const en = document.querySelectorAll('.en');
+  const enCap = document.querySelectorAll('.enCap');
+  for (let i = 0, j = 0, n = 0, l = 0;
+    i < enCap.length, j < en.length, n < EN.length, l < EN_CAPS.length;
+    i += 1, j += 1, n += 1, l += 1) {
+    en[i].innerHTML = EN[i];
+    enCap[i].innerHTML = EN_CAPS[i];
+  }
+  const body = document.querySelector('body');
+  body.addEventListener('keydown', (event) => {
+    event.preventDefault();
+    if (event.altKey && (event.ctrlKey || event.metaKey)) {
+      rus();
+      localStorage.clear();
+      localStorage.setItem('isEng', '1');
+    }
+  });
+}
+function rus() {
```

Comment:
I believe that function `rus` and `eng` should be combined into one. Because these two methods are different only in 3 lines.